docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,231 @@ sidebar_position: 60
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **API Privileges** | Cns.Searchable<br />Datastore.Browse<br />Host.Config.Storage<br />InventoryService.Tagging.CreateTag<br />InventoryService.Tagging.EditTag<br />Network.Assign<br />Sessions.ValidateSession<br />StorageProfile.View<br />StorageViews.View |
 
 </TabItem>
 
 </Tabs>
 
-:::warning
+### Spectro Root Role Assignments
 
-If the network is a Distributed Port Group under a vSphere Distributed Switch (VDS), _ReadOnly_ access to the VDS
-without “Propagate to children” is required.
+The privileges associated with the _Spectro root role_ must be granted via role assignments on specific vSphere objects
+for either the user or a group containing the user. Review the required role assignments to ensure that your user has
+all required privileges on all required objects.
 
-:::
+| **vSphere Object**     | **Propagation** | **Role**            | **Condition**                                                            |
+| ---------------------- | --------------- | ------------------- | ------------------------------------------------------------------------ |
+| **vCenter Root**       | No              | _Spectro root role_ |                                                                          |
+| **Target Datacenter**  | No              | _Spectro root role_ |                                                                          |
+| **Target Cluster**     | No              | _Spectro root role_ |                                                                          |
+| **Target VM Folder**   | Yes             | _Spectro root role_ |                                                                          |
+| **Distributed Switch** | No              | _Spectro root role_ | If the Target Network is a Distributed Port Group residing beneath a VDS |
 
 ### Spectro Role Privileges
 
-As listed in the table, apply spectro role privileges to vSphere objects you intend to use for Palette installation. A
-separate table lists Spectro role privileges for VMs by category.
-
-Open Virtual Appliance (OVA) files are downloaded to the folder you selected. These images are cloned from the folder
-and applied VMs that deployed during deployments.
+The VMware vSphere user account that deploys host clusters and/or private cloud gateways requires all vSphere privileges
+listed in the following table on specific objects. A _Spectro role_ must be created that contains each privilege in the
+following tables. A separate table lists Spectro role privileges for VMs by category.
```

Comment:
```suggestion
A _Spectro role_ must be created that contains each privilege in the following tables. A separate table lists Spectro role privileges for VMs by category.
```

Reply:
Resolved in eff3548d1103ad25895211ffc3fc1076aabc3de1